doc/source/whatsnew/v0.25.0.rst

```diff
@@ -170,7 +170,7 @@ Plotting
 Groupby/Resample/Rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
--
+- Bug in :method:'agg' when pqssing a list in the method agg on a Groupby with attibute as_index set as False, returned a DataFrame with the Groupby key as an index, now return the Groupby key as a column.
```

Comment:
can you make this much simpler, I am not sure what you are fixing here. add the issue number